src/models/ImgixModel.php

```diff
@@ -189,6 +189,10 @@ public function __construct($image, $transforms = null, $defaultOptions = [])
             if ($domain !== null) {
                 $domainParts = explode('/', $domain, 2);
                 $domain = $domainParts[0];
+            } else {
+                // Domain isn't in imgixDomains, just passthrough the image
+                $this->transformed = $image;
```

Comment:
Also had another thought last night, this could be 

```twig
$this->transformed = $image->copyWithTransform($transforms)
```

If you'd want to return the craft transformed asset as well